CMakeLists.txt

```diff
@@ -42,13 +42,16 @@ install(DIRECTORY include/
   DESTINATION include
 )
 
-install(TARGETS DepthImageToLaserScan DepthImageToLaserScanROS depthimage_to_laserscan_node
-  EXPORT export_depthimage_to_laserscan_node
```

Comment:
Hm, good question.  I don't know where that came from.  It seems to build and run without that, so I think it is fine to remove it.